osu.Game/Overlays/ChatOverlayV2.cs

```diff
@@ -311,4 +314,72 @@ private void handleChatMessage(string message)
                 channelManager.PostMessage(message);
         }
     }
+
+    public class ChatOverlayDrawableChannel : DrawableChannel
```

Comment:
`ChildrenOfType<DrawableChannel>()` doesn't seem to work in the visual tests which is why I exposed this class publicly and made the tests use the type explicitly

Reply:
It surely should work as `ChildrenOfType` handles receiving base types and returning subclass-typed instances. Worth debugging what’s going on with that method in that case.

Reply:
I was wrong earlier, it does work.
I was using `ChildrenOfType<Container<DrawableChannel>>().Single().Child` which is redundant since I could just use `ChildrenOfType<DrawableChannel>()` which works as expected.
Additionally, I have moved the `ChatOverlayDrawbleChannel` class into it's own file since it felt it was large enough to warrant it so it has to remand public for now.